modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/select/SelectDDMFormFieldTemplateContextContributor.java

```diff
@@ -172,14 +172,20 @@ protected List<Map<String, String>> getOptions(
 		String objectFieldName = GetterUtil.getString(
 			ddmFormField.getProperty("objectFieldName"));
 
+		List<Map<String, String>> options = new ArrayList<>();
+
 		if ((objectDefinition != null) &&
 			Validator.isNotNull(objectFieldName)) {
 
-			return _getOptionsFromObjectField(
+			options = _getOptionsFromObjectField(
 				objectDefinition.getObjectDefinitionId(), objectFieldName);
+
+			if (options.size() > 0) {
+				return options;
+			}
```

Comment:
@MirnaGama can you check if we can use the `ListUtil.isNotEmpty` instead? `options` can be `null`

Reply:
@carolmariaabb done !